Author: Denis Kristak (Inuits)
Removing the `group` keyword (not supported by current poetry version).
diff -ruN pygam-0.9.0_orig/pyproject.toml pygam-0.9.0/pyproject.toml
--- pygam-0.9.0_orig/pyproject.toml	2023-09-22 13:51:38.904799303 +0100
+++ pygam-0.9.0/pyproject.toml	2023-09-22 13:51:44.440735029 +0100
@@ -12,7 +12,7 @@
 scipy = "^1.10.1"
 progressbar2 = "^4.2.0"
 
-[tool.poetry.group.dev.dependencies]
+[tool.poetry.dev-dependencies]
 pytest = "^7.2.2"
 flake8 = "^6.0.0"
 codecov = "^2.1.12"
